pyccl/cls.py

```diff
@@ -3,121 +3,105 @@
 from .core import check
 import numpy as np
 
-# Mapping between names for tracers and internal CCL tracer types
-tracer_types = {
-    'nc':               const.CL_TRACER_NC,
-    'number_count':     const.CL_TRACER_NC,
-    'wl':               const.CL_TRACER_WL,
-    'lensing':          const.CL_TRACER_WL,
-    'weak_lensing':     const.CL_TRACER_WL,
-    'cmbl':             const.CL_TRACER_CL,
-    'cmb_lensing':      const.CL_TRACER_CL,
-}
-
```

Comment:
Out of curiosity: why did you get rid of these? I'd have thought that users being able to use informative strings rather than constants would be friendlier.

Reply:
No longer needed. You can’t instantiate a tracer directly.

Reply:
You have to use a subclass instead and so I am using the constants directly.